### PR TITLE
[WIP] Improvements to the manual

### PR DESCRIPTION
**Pull request by asinghvi17**
Since I will probably be using `circuitikz` more frequently now, hopefully this will make some substantial progress in the next couple of weeks.  

Pace will be slow, sorry about that :(

### CODE REVIEW

**Comment by asinghvi17** [comment]
@Rmano, what would you think of having hyperlink color be blue or something?  It would be nice to distinguish hyperlinks from regular text.

**Comment by Rmano** [comment]
> @Rmano, what would you think of having hyperlink color be blue or something? It would be nice to distinguish hyperlinks from regular text.

Yes, I'd like that, as long as they are not the "boxed" type that I personally dislike... but blue color is ok.

**Comment by Rmano** [comment]
> Pace will be slow, sorry about that :(

No problem at all. Just ping when you think it's ready for merging. So far, so good ;-)

**Comment by asinghvi17** [comment]
I was also thinking about adding a simple breadboard layout to show how it can be done.  Perhaps a half-breadboard with connections would be adequate.

Are there any general stylistic rules which apply to breadboard designs?  I was thinking that the rows could be bridged by a very light gray line, but I'm not sure how to represent, for example, a resistor above an inline chip.  Perhaps it could be a different colour - but that would make it look bad.

**Comment by asinghvi17** [comment]
Agreed, must be that tabs snuck in somewhere.  Will reformat.

**Comment by asinghvi17** [comment]
@Rmano, do you know if Travis CI is enabled on the repo?  I can't seem to find it.

File: doc/circuitikzmanual.tex [hunk 1]
```diff
@@ -1706,17 +1721,18 @@ \subsection{Electronic Tubes}
...
-	\draw (magn.anode) to [short] (magn.anode|-gnd) node[rground]{};
-	\draw (magn.cathode1)node[above]{$1$};
-	\draw (magn.cathode2)node[above]{$2$};
-	\draw[->](magn.east) --++(1,0)node[right]{$RF_{out}$};
-	\end{circuitikz}
+\begin{circuitikz}
+\draw (0,-2)node[rground](gnd){} to[voltage source,v<={HV}]++(0,3)--++(1,0)to[V,n=DC]++(2,0);
+\draw (2,-1) node[magnetron,scale=1](magn){};
+\draw (DC.left)++(-0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode1);
+\draw (DC.right)++(0.2,0)to [short,*-] ++(0,-1) to [short] (magn.cathode2);
```
Comment (asinghvi17) [inline doc/circuitikzmanual.tex]: I think I've messed up the indentation on these, and I can't seem to restore it.  Other than that, I think that this is in a mergable state.

**Comment by Rmano** [comment]
Seems ok, will merge soon. Thank you!

**Comment by Rmano** [comment]
Thanks a lot!

**Pull request merged**